index.js

```diff
@@ -129,7 +221,20 @@ function filterByGenre() {}
       }
     ];
  */
-function getAllMoviesReleasedAtOrBeforeYear() {}
+function getAllMoviesReleasedAtOrBeforeYear(movies, year) {
+  if(movies.length === 0){
+    return [];
+  }
+
+  let movieObject = [];
+  for (let movie of movies){
+    if(movie.released[2] >= year){
```

Comment:
Take a look here - remember `movie.released` is a string - so indexing `[2]` will give us the second character in the string.